binding_generator.py

```diff
@@ -2569,6 +2569,8 @@ def correct_default_value(value, type_name):
         return f"{type_name}()"
     if value.startswith("Array["):
         return f"{{}}"
+    if value.startswith("&"):
+        return value[1::]
```

Comment:
Unsure if this should instead be:
```suggestion
        return f"StringName({value[1::]})"
```
Not sure what is preferred